src/components/input/DurationInput.js

```diff
@@ -203,6 +208,12 @@ DurationInput.propTypes = {
   error: PropTypes.bool,
   /** In minutes **/
   max: PropTypes.number,
+  size: 'small' | 'medium' | 'large',
```

Comment:
Does this actually work or did you mix up proptypes and typescript? 😅

Reply:
Woops, you're right, I'm not getting any errors but this doesn't seem to be the right way to define this 🙈 Fixed!